test/webserv/integration/common_response.py

```
@@ -40,48 +28,50 @@ def create_response_header(
 )
 response_header_get_root_200_keep = create_response_header(
-    200, "OK", KEEP_ALIVE, root_index_file_length, TEXT_HTML
+    200, KEEP_ALIVE, root_index_file_length, TEXT_HTML
 )
 response_header_get_sub_200_close = create_response_header(
-    200, "OK", CLOSE, sub_index_file_length, TEXT_HTML
+    200, CLOSE, sub_index_file_length, TEXT_HTML
 )
 response_header_400 = create_response_header(
-    400, "Bad Request", CLOSE, bad_request_file_400_length, TEXT_HTML
+    400, CLOSE, bad_request_file_400_length, TEXT_HTML
```

Ruff

Undefined name `bad_request_file_400_length`

(F821)